backend/src/sync.js

```diff
@@ -15,10 +15,16 @@ async function saveBlocks(blocksInfo) {
       try {
         await models.Block.bulkCreate(
           blocksInfo.map((blockInfo) => {
+            let prevHash = blockInfo.header.prev_hash;
+            if (
+              (blockInfo.header.prev_hash = "11111111111111111111111111111111")
+            ) {
+              prevHash = "Genesis";
+            }
```

Comment:
We don't need to do this during sync. It is better to keep the data of the same type. If we store the hash in base58 form, let's keep it that way.